docs/source/analyst/example_usage.ipynb

```diff
@@ -314,23 +402,33 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dl1 = flowclient.daily_location(date=\"2016-01-01\", aggregation_unit=\"admin3\", daily_location_method=\"last\", subscriber_subset=None)\n",
-    "dl2 = flowclient.daily_location(date=\"2016-01-02\", aggregation_unit=\"admin3\", daily_location_method=\"last\", subscriber_subset=None)"
+    "# dl1 = flowclient.daily_location(\n",
```

Comment:
Hmm. Actually, can we either remove this cell, or uncomment it?

Reply:
Good point. I think it's useful to show there are two ways to create a modal location, so I've kept this cell in and uncommented it.